package/lib/compileFunctions.js

```diff
@@ -62,6 +63,19 @@ module.exports = {
         });
       }
 
+      if (funcObject.vpcEgress) {
+        let egress =
+          _.get(funcObject, 'vpcEgress') || _.get(this, 'serverless.service.provider.vpcEgress');
+        if (egress) {
```

Comment:
isn't that check redundant? The whole if will be executed only if `funcObject` has `vpcEgress` set, which means that `egress` will always be set.